gen3/docs/gen3-resources/developer-guide/architecture-new.md

```diff
@@ -0,0 +1,74 @@
+
+
+### Data Model
+
+The Gen3 data model is a graph-like relational model with nodes and edges; it specifies how different files, patients, experiments, and clinical visits are all related to one another.  It is central to a Gen3 commons, which is autogenerated after the data dictionary and data model are specified.  To be more specific, each node represents an entity, which consists of a related group of attributes or data elements. Clinical variables like a cancer diagnosis or a subject’s gender might go into the diagnosis or demographic nodes, respectively. Variables related to how a biological sample was collected or processed may be found in a biospecimen node. Data files, such as medical images or genomic files, can also be nodes and have associated metadata variables like file size, format, and file name.  
+
+Edges or links between nodes indicate relationships between them. For example, a Sample node may derive or connect to a specific patient or a genomic data file may also be associated with a specific sample. Properties are assigned different types including: string, boolean, floating point number, integer, or enumeration. Properties can also be defined as arrays of any of those types. The acceptable values for properties can be further restrained by defining regex patterns that strings must match or minimum or maximum values for numeric data. Nodes, properties, and permissible values are specified in a series of YAML files.
+
+### Data Portal
+
+Since the launch of Gen3, the user interface has been powered by the monolithic Gen3 Data Portal. Its design presents challenges in data movement between pages and has become increasingly difficult to extend. We are developing a new service, the Gen3 Frontend Framework (Gen3.2), to overcome these limitations. It offers enhanced custom content development, application performance, deployment, and maintenance. As we transition to the new framework, we will focus on the Gen3 Frontend Framework rather than the soon-to-be-replaced windmill service.
```

Comment:
I don't particularly like calling the Gen3 Frontend Framework: Gen3.2

It doesn't match any real versioning president we have and imo it muddies the brand name of Gen3. I can see people asking "Wait, so what's the difference? Oh it's just a new frontend, so does the original Gen3 services get replaced, are they compatible?"

I think it might just add confusion

Reply:
Also we need to clarify that windmill == data-portal, or just call it the Gen3 Data Portal like we did in the first sentence

Reply:
I removed Gen3.2.  It will probably appear in some "marketing" material, but it probably help here.  I also removed windmill, which actually doesn't really appear in many place so probably doesn't need to be here.